crane/container.go

```diff
@@ -805,10 +804,6 @@ func (c *container) createArgs(cmds []string, excluded []string) []string {
 	if len(c.RunParams().MemorySwap()) > 0 {
 		args = append(args, "--memory-swap", c.RunParams().MemorySwap())
 	}
-	// MemorySwappiness
-	if c.RunParams().MemorySwappiness > -1 {
```

Comment:
The valid range is 0-100, so we can't simply do `> 0` here since that means people won't be able to use that value... Any idea? I guess we need to wrap that into something similar to `OptBool`...